src/index.html

```diff
@@ -1,14 +1,64 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="html">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0,
+        maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&amp;display=swap"
+      rel="stylesheet"
+    >
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="body">
+    <header class="header">
+
+      <a href="index.html" class="logo_link">
+        <img
+          src="images/logo.png"
+          alt="MOYO-logo"
+          class="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav--list">
+          <li class="nav--list-item">
+            <a href="#" class="is-active">Apple</a>
+          </li>
+          <li class="nav--list-item">
+            <a href="#" class="nav--link">Samsung</a>
```

Comment:
Will be better to specify anchor links, instead of using just # for all cases
```suggestion
            <a href="#samsung" class="nav--link">Samsung</a>
```